src/ui/components/DashKit/plugins/Title/Title.tsx

```diff
@@ -124,60 +149,79 @@ const titlePlugin: PluginTitle = {
             data.text,
         ]);
 
-        const showAnchor = !titlePlugin.hideAnchor && !props.editMode;
-        const withInlineAnchor = showAnchor && isInlineAnchor;
-        const withAbsoluteAnchor = showAnchor && !isInlineAnchor;
-
-        const calculateAnchor = React.useCallback(() => {
+        const calculateExtraElements = React.useCallback(() => {
             if (contentRef.current && rootNodeRef.current) {
                 const contentRect = contentRef.current.getBoundingClientRect();
                 const rootRect = rootNodeRef.current.getBoundingClientRect();
 
                 const offsetTop = contentRect.top - rootRect.top;
 
-                const isWidthFits = contentRect.width + MAX_ANCHOR_WIDTH < rootRect.width;
-                const isHeightFits = contentRect.height <= rootRect.height;
+                const isWidthFits = !isTitleOverflowed(
```

Comment:
Better check `extraRef.current` and `contentRef.current.children[0]` not to be null